src/aliases/manager/service.go

```diff
@@ -1,50 +1,19 @@
 package aliasmanager
 
 import (
-	"context"
-
-	aliasent "github.com/consensys/quorum-key-manager/src/aliases/entities"
-	aliasstore "github.com/consensys/quorum-key-manager/src/aliases/store"
+	"github.com/consensys/quorum-key-manager/pkg/app"
+	aliaspg "github.com/consensys/quorum-key-manager/src/aliases/store/postgres"
+	"github.com/consensys/quorum-key-manager/src/infra/postgres"
 )
 
-type BaseManager struct {
-	db aliasstore.Database
-}
-
-func New(db aliasstore.Database) *BaseManager {
-	return &BaseManager{
-		db: db,
+// RegisterService creates and register the alias service in the app.
+func RegisterService(a *app.App, pgClient postgres.Client) error {
+	db := aliaspg.NewDatabase(pgClient)
```

Comment:
As we are discussing on the other PR(https://github.com/ConsenSys/quorum-key-manager/pull/253/files#diff-f6c71e04a55bbc1dbf8f31101336e0ac7589dabdd7a654c60a00964281bd0443R22) I would rather create a new client for each domain so we keep the intended independency between services

Reply:
done